projects/styles/markup/tui-space.less

```diff
@@ -36,7 +36,7 @@
     }
 }
 
-.tui-space {
+.tui-space.tui-space {
```

Comment:
That wouldn't work as it would generate `.tui-space.tui-space_horizontal-x`. We need to update line 6 instead

Reply:
Updated